lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -289,13 +293,27 @@ struct ConvertUnaryOp : ConversionPattern {
           Value result;
           if (isa<AtenTanhOp>(op)) {
             result = b.create<math::TanhOp>(loc, args[0]);
+          } else if (isa<AtenReluOp>(op)) {
+            result = createScalarRelu(b, loc, args);
           }
           b.create<linalg::YieldOp>(loc, result);
         });
-
     return success();
   }
 };
+
+Value static createScalarRelu(OpBuilder &b, Location loc, ValueRange args){
+  auto elementType = args[0].getType();
+  // TODO: Add support for integer types.
+  assert(elementType.isa<::mlir::FloatType>() &&
```

Comment:
Can you install clang-format? It has pretty good vim and vscode integration. It is a huge time saver :)

Reply:
Thanks for the suggestion!